docs: update AWS Batch service (#183)
localstack/localstack-docs: src/content/docs/aws/services/batch.mdx

```bash
awslocal iam create-role \
    --role-name myrole \
    --assume-role-policy-document '{
  "Version": "2025-10-17",
  "Statement": [
    {
      "Effect": "Allow",
      "Principal": {
        "Service": "ecs-tasks.amazonaws.com"
      },
      "Action": "sts:AssumeRole"
    }
  ]
}'
```

Then attach the ECS task execution policy:

```bash
awslocal iam attach-role-policy \
    --role-name myrole \
    --policy-arn arn:aws:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy
```

### Create the compute environment

You can use the [`CreateComputeEnvironment`](https://docs.aws.amazon.com/cli/latest/reference/batch/create-compute-environment.html) API to create a compute environment.

Run the following command using the role ARN above (arn:aws:iam::000000000000:role/myrole) to create a managed compute environment with FARGATE:

```bash
awslocal batch create-compute-environment \
    --compute-environment-name myenv \
    --type MANAGED \
    --state ENABLED \
    --compute-resources type=FARGATE,maxvCpus=128,subnets=subnet-12345678,securityGroupIds=sg-12345678 \
    --service-role arn:aws:iam::000000000000:role/myrole
```

:::note

> While networking resources such as subnets and security groups are required as input, LocalStack does not create real cloud infrastructure. These values must still be present but are handled within the local ECS runtime.

Reviewer: ```suggestion
While networking resources such as subnets and security groups are required as input, LocalStack does not create real cloud infrastructure. These values must still be present for the compute environment to be created.
```
